rllib/core/models/torch/base.py

```diff
@@ -104,3 +106,12 @@ def get_num_parameters(self) -> Tuple[int, int]:
             num_trainable_params,
             num_all_params - num_trainable_params,
         )
+
+    @override(Model)
+    def _set_to_dummy_weights(self, value_sequence=(-0.02, -0.01, 0.01, 0.02)):
```

Comment:
What is the mechanism to ensure the order of parameters are the same between frameworks?

Reply:
I'm not sure either. For torch, it's neither the order, in which you define Parameter properties in the ctor, nor alphabetical. Didn't have time to investigate.